### DISCUSSION TITLE
Just a quick thank you

### DISCUSSION THREAD

**Discussion by danielfdickinson** [creativeprojects/resticprofile#384] [General]
As the title says. I used to have some homegrown scripts for Windows and Linux (and was anticipating needing one for MacOS) but that's all been replaced with resticprofile now (and I don't have to figure out how to do all this on MacOS). Kudos!

BTW if it would be useful I could write some info on using icacls.exe in Windows to stash the profiles.toml (or profiles.yaml, ...) in a directory only accessible by the Administrators group while having logs that are read-only for a particular user. I do that to reduce exposure of the secrets in profiles.toml and password-file.

**Comment by creativeprojects** [comment]
Ho thank you very much 😊 

Yes please 🙏🏻 
As a matter of fact I'm barely using Windows any more...

You can add it directly to the documentation via a Pull Request if you wish: it's simply written in markdown (inside `docs/content`)

**Comment by danielfdickinson** [comment]
FYI, I expect to have time to work on this in 2-3 weeks.
